attest/tpm_linux.go

```diff
@@ -116,7 +116,7 @@ func openTPM(tpm probedTPM) (*TPM, error) {
 			interf = TPMInterfaceKernelManaged
 		}
 
-		rwc, err = tpm2.OpenTPM(devPath)
+		rw, err = tpm2.OpenTPM(devPath)
```

Comment:
This TPM is never closed. Please revert the s/io.ReadWriteCloser/io.ReadWriter/g changes. You can make Close no-op if you want

```
type noOpCloser struct {
    io.ReadWriter
}
func (n noOpCloser) Close() error {
    return
}
```